components/camel-huaweicloud-smn/src/test/resources/testconfiguration.properties

```diff
@@ -0,0 +1,6 @@
+## Test configurations
```

Comment:
We need to add the Apache License Header here.

Reply:
License header added